src/ast/passes/resource_analyser.cpp

```diff
@@ -289,6 +302,41 @@ void ResourceAnalyser::visit(Call &call)
       resources_.str_buffers++;
   }
 
+  if (call.func == "count" || call.func == "sum" || call.func == "max" ||
```

Comment:
Curious, why not `hist` and `lhist` ?

Reply:
Oops I forgot them since they use getMultiMapKey / getHistKey, I have fixed in latest version